frontend/shared/src/components/toast/show-success-toast.ts

```diff
@@ -0,0 +1,11 @@
+import Toast from 'shared/components/toast/Toast';
```

Comment:
Another option would be to rename current `show-error-toast` to `error-toast` and add `type` as required parameter, so we could use same component for `type: 'error'` and `type: 'succes'`. Of course it needs some refactoring which can be daunting job...

Reply:
I also thought about using multi purpose component but chose the easy way instead and used the same convention that was used of the show-success